rascal2/ui/view.py

```diff
@@ -69,3 +82,22 @@ def createStatusBar(self):
         """Creates the status bar"""
         sb = QtWidgets.QStatusBar()
         self.setStatusBar(sb)
+
+    def setupMDI(self):
+        """Creates the multi-document interface"""
+        widgets = {
+            "Plots": self.plotting_widget,
+            "Project": self.project_widget,
+            "Terminal": self.terminal_widget,
+            "Fitting Controls": self.controls_widget,
+        }
+
+        for title, widget in reversed(widgets.items()):
+            widget.setWindowTitle(title)
+            window = self.mdi.addSubWindow(
+                widget, QtCore.Qt.WindowType.WindowMinMaxButtonsHint | QtCore.Qt.WindowType.WindowTitleHint
+            )
+            window.setWindowTitle(title)
+        # TODO implement user save for layouts, this should default to use saved layout and fallback to tile
+        self.mdi.tileSubWindows()
```

Comment:
not worth doing if it's particularly difficult (only makes an aesthetic change) but in RasCAL-1, the Tile windows button doesn't do fully symmetric tiling - the terminal and controls widgets are shorter than the plotting and controls